docs/UserGuide.adoc

```diff
@@ -316,11 +316,39 @@ e.g. 24:00 is not allowed.
 these tasks cannot be in the same time period, for example, in the same day.
 ****
 
-==== Viewing all tasks
+==== Editing a task
+
+Edits the information of a task. +
+Format: `task edit MOD_CODE ID_NUMBER [/desc DESCRIPTION] [/on DATE/non] [/at TIME]` +
+Example: `task edit CS2103T 848 /desc UG submission /on 12/04/2020 /at 23:59` or `task edit CS2103T 848 /on non`
+
+==== Deleting a task
+
+Deletes a task from the module. +
+Format: `task delete /code MOD_CODE /task TASK_NAME` +
+Example: `task delete /code CS2103T /task IP week 4`
```

Comment:
Editing and deleting task is already in the UG below finding upcoming task

Reply:
Resolved. Thanks!